geointel/config.py

```
@@ -0,0 +1,29 @@
+DEFAULT_MIME_TYPE = "image/jpeg"
+
+# Default generation configuration for Gemini API
+DEFAULT_GENERATION_CONFIG = {
+    "temperature": 0.4,
+    "topK": 32,
+    "topP": 1,
+    "maxOutputTokens": 2048
+}
+
+
+def get_api_key(api_key: str = None) -> str:
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)